packages/plugin-data/src/commands/force/data/bulk/delete.ts

```diff
@@ -46,7 +46,7 @@ export default class Delete extends DataCommand {
       const conn: Connection = this.org.getConnection();
       this.ux.startSpinner('Bulk Delete');
 
-      const csvRecords: ReadStream = fs.createReadStream(this.flags.csvfile);
+      const csvRecords: ReadStream = fs.createReadStream(this.flags.csvfile, { encoding: 'utf-8' });
```

Comment:
@WillieRuemmele, I know utf-8 is very common, but is it exclusive?

Reply:
I believe it is the standard for the CLI

Reply:
What I am concerned about is a user that creates a CSV file **_outside_** of cli with an encoding that is not utf-8. Chances are slim, but Murphy is always at the ready :).

Reply:
The default for fs.readStream is `null` which means "don't try to deal with encoding at all."  Presumably, those other encodings fail today.

From testing, it **looks** like files with no encoding work fine with utf8.  

[for context, the csv that led to this (utf8 with byte order mark) is generated by Tableau's tooling]

Reply:
Given this particular use case, should the commands be extended to expose file encoding, defaulting to utf-8? Maybe with a user story to, instead of this bug fix?

Is there any error telemetry data that shows the kind of errors encountered to see if adding an option might be required?

Reply:
we also officially say that csv files must be in utf-8 https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/datafiles_csv_preparing.htm